Hush/Assets/Prefabs/Environment/Domes/Relic Dome.prefab

```diff
@@ -0,0 +1,142 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!1 &4256472995991901031
+GameObject:
```

Comment:
Since the relic dome will always have a relic, could be good to add the relic into the prefab directly